macos/Podfile.lock

```diff
@@ -1769,65 +1769,65 @@ SPEC CHECKSUMS:
+  React-perflogger: 3756bfe3ff8cc8fffed3c51dce34cef861d4b82a
+  React-performancetimeline: c4678b2226a660268cc478662ffc43a2ab1b5d30
   React-RCTActionSheet: 411a6257051d5b24dc66704851cc9235f958e65a
-  React-RCTAnimation: c67f4637f01a3e2079afba0a55cb522eb5156076
-  React-RCTAppDelegate: 8d6486592b38c40191b548b84d64a2a6c9d44fad
-  React-RCTBlob: 222542e95b2b3737cf0a3ec5c96718ade8a5db15
-  React-RCTFabric: 1bc56590f16fee228398653874c377a334cbd6a0
-  React-RCTFBReactNativeSpec: 99ab0fb589bb70f02f6e1b158e1dc137d3c80453
-  React-RCTImage: 7f252b669d1a9a730a0f43f08e1af2d54129fc40
-  React-RCTLinking: e0860ec9f132e1d3701fd065076e730003fbc7d0
-  React-RCTNetwork: c604de4daff9e8c2e9171f82b5e7b6100d7b4f08
-  React-RCTSettings: d0540aeff36109b4caf2fc871d0f8558a0805a97
-  React-RCTText: db7e7bf3811ed182b81ef2458214c7aa8992f936
-  React-RCTVibration: 1c291931789c0803b9429f94c5d7a2e215aba3f3
+  React-RCTAnimation: c0197d1545a45ade3517f34271ec9c81b1aee5fd
+  React-RCTAppDelegate: c94c03ef7dfad0a446c4ecff6dc9252860cfc80e
+  React-RCTBlob: 23fd27a68f1f28fb3f3abf7b22a429a4d65fd9d9
+  React-RCTFabric: cccaa2f386b67512c80fdaf3e548d6fb7bf4c1e8
+  React-RCTFBReactNativeSpec: 99c54bd15b054a509b0fdc960f222b2683ab4fb2
+  React-RCTImage: c00be431cd5639cb63241d427ebb14593a82e5f7
+  React-RCTLinking: 0b0399a963b358af083c2b37a10ec542bfe7cd5d
+  React-RCTNetwork: ca3adfccdccf3898839832964fabea7106b7cd78
+  React-RCTSettings: f0df90a9db805c2a291d67ec0af0fbb6b1cb8f0e
+  React-RCTText: 6111a0d5fbd69455b99d710aaf2fb2e29d1d3a3a
+  React-RCTVibration: 93ac072f90ecc83871eac50ad6c51738d37e590e
   React-rendererconsistency: 007dccdd75e782d49834c68f0c412449048d40e4
-  React-rendererdebug: bab289a7c248c61d41f5f317013c1fe0dae467ad
+  React-rendererdebug: 019dc752d5de90475252347cc3d7897cd2a22521
   React-rncore: a8ffa6d315742a2bf13e628591368453dfdc3657
-  React-RuntimeApple: beb34db74a1b9d08634609c89b78e0e2f366ab27
-  React-RuntimeCore: e8c71261118ef30213f9b8da6206d953700964e3
+  React-RuntimeApple: 9e5dcaa897cd16aeadfaeb7ced8ea4dd77926195
+  React-RuntimeCore: c4ab914b0498eda39f17d03618669f7572c4509c
   React-runtimeexecutor: f7745ae40c91165e12293bdf48889ed85266b5e6
-  React-RuntimeHermes: a6aaca3dd31c834a3a1e02a8b587eaa97e895455
-  React-runtimescheduler: 18bdf3609267412f28f8c72c963307f3f4200a57
+  React-RuntimeHermes: 1f676783fb0b92bd75035d08fbab3abb06c5d057
+  React-runtimescheduler: ef1542b69b0fdd1121b430081c0617c93fbafe74
   React-timing: 603df0b59afb508934070c569cc568ef72778420
-  React-utils: 4251de6f3e8e2fee8c763b3972c12d1de8143089
-  ReactAppDependencyProvider: dda2eca447ef13387ed59b9605382e036858c801
-  ReactCodegen: f82c403b99b908a5e3a11fa2db9dbfe402958e41
-  ReactCommon: 06d6013a960cfd9223a4f6d335343ce04c0edbbe
+  React-utils: 272bde9a8aa4037b23e2c9bf4b44e9a9a6cbdfaf
+  ReactAppDependencyProvider: a12262458b50521ba56afb93f4cc875732f9d643
+  ReactCodegen: 2b6c7d7db8ed68c1ea03196967f3e7b389d9df7b
+  ReactCommon: 6da1953722466adfd0f0bdf5eb5f5551ff53aae1
```

Comment:
@leonskim @SeanBarker182  Should we remove the Podfile.lock from version control? This is going to get very annoying very quickly.

Reply:
what does `--deployment` do?